aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -83,8 +93,10 @@ module aptos_std::ristretto255_bulletproofs {
         )
     }
 
-    /// Verifies a zero-knowledge range proof that the value `v` committed in `com` (as v * val_base + r * rand_base,
-    /// for some randomness `r`) satisfies `v` in `[0, 2^num_bits)`. Only works for `num_bits` in `{8, 16, 32, 64}`.
+    /// Verifies a zero-knowledge range proof for a batch of commitments `comms` (each of the form 
```

Comment:
accidentally copied from the batch version?

Reply:
Thanks, fixed